addons/account_edi_ubl_cii/tests/test_ubl_cii.py

```diff
@@ -378,3 +380,63 @@ def test_import_discount(self):
         imported_invoice = self.import_attachment(xml_attachment, self.company_data["default_journal_sale"])
         for line in imported_invoice.invoice_line_ids:
             self.assertFalse(line.discount, "A discount on the imported lines signals a rounding error in the discount computation")
+
+    def test_export_xml_with_multiple_invoices(self):
+        partner = self.env['res.partner'].create({
+            'name': 'Test Partner',
+            'invoice_edi_format': 'ubl_bis3',
+            'country_id': self.env.ref('base.be').id,
+        })
+        invoice_1, invoice_2, invoice_3 = self.env['account.move'].create([
+            {
+                'partner_id': partner.id,
+                'move_type': 'out_invoice',
+                'invoice_line_ids': [
+                    Command.create({
+                        'product_id': self.product_a.id,
+                        'quantity': 1,
+                        'price_unit': 100,
+                    }),
+                ],
+            },
+            {
+                'partner_id': partner.id,
+                'move_type': 'out_invoice',
+                'invoice_line_ids': [
+                    Command.create({
+                        'product_id': self.product_a.id,
+                        'quantity': 2,
+                        'price_unit': 200,
+                    }),
+                ],
+            },
+            {
+                'partner_id': partner.id,
+                'move_type': 'out_invoice',
+                'invoice_line_ids': [
+                    Command.create({
+                        'product_id': self.product_a.id,
+                        'quantity': 3,
+                        'price_unit': 300,
+                    }),
+                ],
+            },
+        ])
```

Comment:
```suggestion
        invoices = self.env['account.move'].create([
            {
                'partner_id': partner.id,
                'move_type': 'out_invoice',
                'invoice_line_ids': [
                    Command.create({
                        'product_id': self.product_a.id,
                        'quantity': qty,
                        'price_unit': price,
                    }),
                ],
            }
            for qty, price in [(1, 100), (2, 200), (3, 300)]
        ])
```
Should use qty and amount in loop since another values are same.

Reply:
Done